modules/installation-configuration-parameters.adoc

```diff
@@ -1000,7 +1000,11 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Number
 
 |`<machine-pool>.platform.ovirt.vmType`
-|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.
+|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.  By default, master nodes use `high-performance`, and worker nodes use `server`. For details, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Virtual_Machine_General_settings_explained[Explanation of Settings in the New Virtual Machine and Edit Virtual Machine Windows] in the _Virtual Machine Management Guide_.
+[NOTE]
+====
+`high_performance` improves performance on the VM, but you cannot access the VM with a console, and the Administration Portal does not show any memory statistics for the node.
```

Comment:
I took it from the RHV docs linked above
```
high performance virtual machine has certain limitations because enhanced performance has a trade-off in decreased flexibility, for example, the VM will not be accessible via a graphical console, for more details review the RHV documentation.
```

Reply:
@Gal-Zaidman  The document you linked to above does not say anything about a graphical console. Do we need to add info about a graphical console not being available to that RHV topic?

Reply:
Updated.